zulip_bots/zulip_bots/bots/jira/jira.py

```diff
@@ -127,7 +163,40 @@ def initialize(self, bot_handler: Any) -> None:
             raise KeyError('No `domain` was specified')
 
         self.auth = make_jira_auth(username, password)
-        self.domain_with_protocol = 'https://' + domain
+
+        # Allow users to override the HTTP scheme
```

Comment:
It would be better to split this into its own commit. I'm not sure if this a good change; we should try to encourage using https and not provide an easy way to override that.